Sources/Conduit/Auth/OAuth2RequestPipelineMiddleware.swift

```diff
@@ -163,9 +163,10 @@ public struct OAuth2RequestPipelineMiddleware: RequestPipelineMiddleware {
                                     password: clientConfiguration.clientSecret)
 
         let requestBuilder = HTTPRequestBuilder(url: clientConfiguration.environment.tokenGrantURL)
-        requestBuilder.bodyParameters = ["grant_type": "refresh_token",
-                                         "refresh_token": refreshToken,
-                                         "scope": clientConfiguration.environment.scope]
+        var bodyParameters = ["grant_type": "refresh_token",
+                              "refresh_token": refreshToken]
+        bodyParameters["scope"] = clientConfiguration.environment.scope
```

Comment:
Might want to add a comment before this line mentioning it wont be set if `clientConfiguration.environment.scope` is `nil`.